### ISSUE TITLE
World tides

### ISSUE THREAD

**Issue by lukehefson** [jpr5/webcaltides#28]
This utility looks amazing! I'd love to use it, but unfortunately I'm not often in the US or Canada. It would be amazing if it supported worldwide tides.

However, from cursory googling– perhaps that requires a paid API? In which case I can appreciate if you don't want to support it and close this issue!

**Comment by jpr5** [comment]
Yeah I would love to add in more and support it -- but all of the other services that have that data, have it (1) paywalled and (2) don't provide much of a forecast -- many will limit to a week or a month ahead.  

The only solution I can see would be to take the old data files (that used to be public, and can still be gotten) -- but they're old and the setup/usage of them defies even my own understanding.  It's what these other vendors must have done, but... it's honestly tough to understand how to do it, and no one has simplified their usage in public forums.  

I wish there was more I could do... until I figure the data files out, it's just gonna be US and Canada.  Sorry.  🤷

**Comment by lukehefson** [comment]
Since creating this issue I actually tried something similar myself [here](https://github.com/lukehefson/low-tides-to-ics) using a free WorldTides API key from https://www.worldtides.info/developer.

However, the tide data seemed savagely wrong to me. So I quickly gave up.

Very fair that you don't want to hack around supporting outside of US and Canada.

**Comment by jpr5** [comment]
No no -- I _do want_ to hack around with it!  I just haven't been able to figure out to use the harmonics data files.  It's surprisingly poorly documented, despite there being reference code for it.  

And yes, I suspect it would be wrong anyway -- the harmonics files have been closed-source and so there aren't any publicly-available updates that I think would be needed.  Supports my view that the commercial vendors are using the old harmonics files..

I'd set it up anyway though, if I could figure it out.
